features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -143,8 +182,24 @@ internal class JankStatsActivityLifecycleListener(
     override fun onFrame(volatileFrameData: FrameData) {
         val durationNs = volatileFrameData.frameDurationUiNanos
         if (durationNs > 0.0) {
-            val frameRate = (ONE_SECOND_NS / durationNs)
-            if (frameRate in VALID_FPS_RANGE) {
+            var frameRate = (ONE_SECOND_NS / durationNs)
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                screenRefreshRate =
+                    ONE_SECOND_NS / DDFrameMetricsListener.DEFAULT.frameDeadline
+            } else if (Build.VERSION.SDK_INT == Build.VERSION_CODES.R) {
+                screenRefreshRate = display?.refreshRate?.toDouble() ?: SIXTY_FPS
+            }
+
+            frameRate *= (SIXTY_FPS / screenRefreshRate)
+
+            // If normalized frame rate is still at over 60fps it means the frame rendered
+            // quickly enough for the devices refresh rate.
+            if (frameRate > MAX_FPS) {
+                frameRate = MAX_FPS
+            }
```

Comment:
very minor, just looks shorter

```suggestion
            // If normalized frame rate is still at over 60fps it means the frame rendered
            // quickly enough for the devices refresh rate.
            frameRate *= (SIXTY_FPS / screenRefreshRate).coerceAtMost(MAX_FPS)
```